### ISSUE TITLE
[Bug] Okta MFA Verification requirement coming up each time command is run

### ISSUE THREAD

**Issue by pranav-bhatt** [Nike-Inc/gimme-aws-creds#433]
Each time I run `gimme-aws-creds`, the Okta MFA Verification is required by me each time I run the command.

## Expected Behavior

In the past, it would remember that I have performed MFA already, and would remain authenticated at least for a few hours.

## Current Behavior

I have to perform MFA each time I run the command. Okta doesn't remember that I've already authenticated. The same issue doesn't happen with my browser, where Okta remembers that I've logged in.

## Your Environment

* App Version used: 2.7.2
* Environment name and version: Macbook Pro M1
* Operating System and version: macOS 13.4

**Comment by pranav-bhatt** [comment]
The `-m` falg has no effect

**Comment by anukrati1507** [comment]
I am facing the same issue. The flag seems to be not working as expected.

**Comment by pranav-bhatt** [comment]
@epierce any idea about this? If you need any extra info, I'll be happy to provide the same :)

**Comment by epierce** [comment]
Have you run `gimme-aws-creds --register-device`? That will set the `device_token` value that will be sent to Okta in the DT token.  The `-m` flag doesn't do anything without the device token.

**Comment by pranav-bhatt** [comment]
Yes I have tried with that flag. Still asks for MFA each time. Another thing I noticed is that my device shows up as 'Unrecognised Device' each time now in the MFA prompt. Earlier is used to show up as 'Mac OS...' or something similar.

**Comment by kylefuhrmanncalm** [comment]
Adding on here: If I don't have `device_token` in the config file, using `-m` for me appropriately confirms that I registered a new device, and no longer has `Unrecognized Device` on my Okta Verify prompt, but it does still ask me to verify each login.

So it seems in my example, it's appropriately at least recognizing that its sending the device token.
